tahoma_api/client.py

```diff
@@ -155,6 +156,9 @@ async def __make_http_request(
                 ) as response:
                     result = await response.json()
 
+        # TODO replace by our own library
+        result = humps.decamelize(result)
+
```

Comment:
Because you would need to do it for all keys in a dict, thus requiring loops and more logic. Why not depend on a package that is made for this?

Reply:
Because you can simply add a simple method instead of adding a full dependency for one line:
```python
def camel_to_snake_case(d):
  pattern = re.compile(r'(?<!^)(?=[A-Z])')
  return {pattern.sub('_', k).lower(): v for k, v in d.items()}
```

```
>>> camel_to_snake_case({"ABC": 3})
{'a_b_c': 3}
```
See https://stackoverflow.com/a/1176023